drivers/hmis_external_apis/app/models/hmis_external_apis/external_forms/form_submission.rb

```diff
@@ -76,6 +76,10 @@ def form_values
       raw_data.reject { |key, _| EXTRANEOUS_KEYS.include?(key) }
     end
 
+    def data_collection_stage
+      1 # use Intake for external forms
+    end
```

Comment:
Add a note about usage, since it's not obvious where this is called
```suggestion
    # Consider external form submissions to have "Project Entry" data collection stage, because they may generate new Enrollments. This method is used by the form processor to set DataCollectionStage on related records that are produced, such as Disability records.
    def data_collection_stage
      1
    end
```